### ISSUE TITLE
Inaccurate documentation in "cd-on-quit" script

### ISSUE THREAD

**Issue by pcapriotti** [jarun/nnn#1336]
#### Environment details (Put `x` in the checkbox along with the information)

- [x] Operating System: linux
- [x] Terminal Emulator: alacritty
- [x] Shell: zsh
- [x] Program options used: `-Rdx`
- [x] Issue exists on `nnn` master

#### Exact steps to reproduce the issue

[This sentence](https://github.com/jarun/nnn/blob/master/misc/quitcd/quitcd.bash_zsh#L12) in the function for "cd on quit" is not accurate. If `NNN_TMPFILE` is set to a custom path, but not exported, `nnn` won't see it, so the part of the script after the `nnn` invocation won't see the updated `.lastd` file.

So basically custom paths only work for a "cd on quit always" setup, not for "cd on Ctrl+G".

**Comment by jarun** [comment]
Can you suggest something more accurate?

**Comment by pcapriotti** [comment]
> Can you suggest something more accurate?

I suppose just removing that line would work. And similar ones from the scripts for other shells of course. I can make a PR if it's helpful.

**Comment by jarun** [comment]
@N-R-K sounds good?

**Comment by N-R-K** [comment]
I think I remember getting confused by it as well back when starting out.

> I suppose just removing that line would work.

That sounds good to me. Feel free to open the PR.

But I think we should clarify the custom path part as well. So I'm thinking something like this:

```diff
     # The behaviour is set to cd on quit (nnn checks if NNN_TMPFILE is set)
-    # To cd on quit only on ^G, either remove the "export" as in:
+    # If NNN_TMPFILE is set to a custom path, it must be exported for nnn to see.
+    # To cd on quit only on ^G, remove the "export" and set NNN_TMPFILE *exactly* as this:
     #    NNN_TMPFILE="${XDG_CONFIG_HOME:-$HOME/.config}/nnn/.lastd"
-    #    (or, to a custom path: NNN_TMPFILE=/tmp/.lastd)
-    # or, export NNN_TMPFILE after nnn invocation
     export NNN_TMPFILE="${XDG_CONFIG_HOME:-$HOME/.config}/nnn/.lastd"
```

**Comment by jarun** [comment]
@N-R-K please update the Wiki if required.

**Comment by jarun** [comment]
:+1:
